### ISSUE TITLE
Book 05: Sentence 007

### ISSUE THREAD

**Issue by ftyers** [ftyers/UD_Classical_Nahuatl-FloCo#28]
![imatge](https://github.com/ftyers/UD_Classical_Nahuatl-FloCo/assets/449545/0ec735e7-85e2-4407-b12e-66d4759c7df2)

```
# sent_id = Book_05_-_The_Omens.txt:7
# text = Yn jqc oiuh ipan muchiuh, y: njman ic iauh, qujttaz in tonalpouhquj: vmpa quellaquaoa, qujtlapaloa: qujlvia.
# text[norm] = In ihcuac oyouh ipan mochiuh, y: niman ic yauh, quittaz in tonalpouhqui: ompa quellacuahua, quitlahpaloa: quilhuia.
# text[orig] = Y·njqc¶oiuh·ipan·muchiuh·,·y·:·njman·ic·iauh·,·qujttaz·intonalpouhquj·:·vmpa·quella¶quaoa·,·qujtlapaloa·:·qujlvia·.
# text[osp] = Aujendo oydo este mal aguero: luego yua a buscar, a aquellos que sabian declarar estos agueros: a los quales llamauan tonalpouhque. // I este aguero, o adiuino consolava, y esfurçaua a este tal: diziendole desta manera.
# text[spa] = Cuando eso pasó, entonces iba a ver el tonalpouhqui, donde éste le consolaba, le esforzaba, decía:
# references = A&D/161:1
# tagged = 100.00%
# lemmatised = 100.00%
# analysed = 100.00%
# heads = 100.00%
# relations = 52.17%
1	Yn	in	DET	_	_	3	_	_	Orig=Y·njqc¶|Folio=1v,1v|Paragraph=5,5|Line=14,14|Norm=In
2	jqc	ihcuac	ADV	_	_	1	_	_	Orig=Y·njqc¶|Folio=1v,1v|Paragraph=5,5|Line=14,14|Norm=ihcuac|Gloss=en.el.momento
3	oiuh	oyohua	VERB	_	Number[subj]=Plur,Sing|Person[subj]=3|Subcat=Intr|Tense=Past|VerbForm=Fin	11	_	_	Folio=1v|Paragraph=5|Line=15|Norm=oyouh|Analysed=Yes
4	ipan	pan	NOUN	_	NounType=Relat|Number[psor]=Sing|Person[psor]=3|State=Pos	5	_	_	Folio=1v|Paragraph=5|Line=15|Norm=ipan|Analysed=Yes
5	muchiuh	chihua	VERB	_	_	11	_	_	Folio=1v|Paragraph=5|Line=15|Norm=mochiuh
6	,	,	PUNCT	_		11	punct	_	Folio=1v|Paragraph=5|Line=15|Norm=,|Analysed=Yes
7	y	y	CCONJ	_	_	11	_	_	Folio=1v|Paragraph=5|Line=15|Norm=y|AmbigNorm=True|Override=False
8	:	:	PUNCT	_		11	punct	_	Folio=1v|Paragraph=5|Line=15|Norm=:|Analysed=Yes
9	njman	niman	ADV	_		11	_	_	Folio=1v|Paragraph=5|Line=15|Norm=niman|Analysed=Yes
10	ic	ic	ADV	_	_	11	_	_	Folio=1v|Paragraph=5|Line=15|Norm=ic
11	iauh	ya	VERB	_	Number[subj]=Plur,Sing|Person[subj]=3|Subcat=Intr|Tense=Pres|VerbForm=Fin	0	root	_	Folio=1v|Paragraph=5|Line=15|Norm=yauh|Analysed=Yes
12	,	,	PUNCT	_		13	punct	_	Folio=1v|Paragraph=5|Line=15|Norm=,|Analysed=Yes
13	qujttaz	itta	VERB	_	Number[obj]=Sing|Number[subj]=Plur,Sing|Person[obj]=3|Person[subj]=3|Subcat=Tran|Tense=Fut|VerbForm=Fin	11	_	_Folio=1v|Paragraph=5|Line=16|Norm=quittaz|Analysed=Yes
14	in	in	DET	_	_	15	det	_	Orig=intonalpouhquj|Folio=1v|Paragraph=5|Line=16|Norm=in
15	tonalpouhquj	tonalpouhqui	NOUN	_	State=Abs	13	obj	_	Orig=intonalpouhquj|Folio=1v|Paragraph=5|Line=16|Norm=tonalpouhqui|Analysed=Yes
16	:	:	PUNCT	_		18	punct	_	Folio=1v|Paragraph=5|Line=16|Norm=:|Analysed=Yes
17	vmpa	ompa	ADV	_		18	advmod	_	Folio=1v|Paragraph=5|Line=16|Norm=ompa|Gloss=allá|Analysed=Yes
18	quellaquaoa	ellacuahua	VERB	_	Number[obj]=Sing|Number[subj]=Plur,Sing|Person[obj]=3|Person[subj]=3|Subcat=Tran|Tense=Pres|VerbForm=Fin	11	_	_	Orig=quella¶·quaoa|Folio=1v,1v|Paragraph=5,5|Line=16,17|Norm=quellacuahua|Analysed=Yes
19	,	,	PUNCT	_		20	punct	_	Folio=1v|Paragraph=5|Line=17|Norm=,|Analysed=Yes
20	qujtlapaloa	tlahpaloa	VERB	_	Number[obj]=Sing|Number[subj]=Plur,Sing|Person[obj]=3|Person[subj]=3|Subcat=Tran|Tense=Pres|VerbForm=Fin	18	conj	_	Folio=1v|Paragraph=5|Line=17|Norm=quitlahpaloa|Analysed=Yes
21	:	:	PUNCT	_		22	punct	_	Folio=1v|Paragraph=5|Line=17|Norm=:|Analysed=Yes
22	qujlvia	ilhuia	VERB	_	_	11	_	_	Folio=1v|Paragraph=5|Line=17|Norm=quilhuia
23	.	.	PUNCT	_		11	punct	_	Folio=1v|Paragraph=5|Line=17|Norm=.|Analysed=Yes

```

**Comment by ftyers** [comment]
Questions:
- What on earth is *y* there? 
- What is the structure of *oyouh ipan mochiuh*? 
  - Does *ipan mochiuh* go with *yauh* or with *oyouh*?
- What is the structure of *yauh quittaz*, is it an adverbial clause (there is no *in*) or is it `parataxis`? There are examples in other sentences of both `yauh in [^ ]+z` and `yauh [^ ]+z`, e.g. *niman ye ic yauh in maahuiltiz*

**Comment by Lguyogiro** [comment]
I think _yauh, quittaz_ should be `advcl`. It isn't the yauh + purposive construction that Andrews talks about, but it seems to work similarly ("go to do X"). The fact that there is a comma interposing them makes me a little hesitant, but I'm not sure we can use punctuation consistently as a diagnostic.

**Comment by Lguyogiro** [comment]
> What on earth is y there?

not sure, but I want to think it isn't the Spanish conj. need to research this a bit more.


re ipan mochiuh, I think this goes with oyouh, and is a similar construction as the yauh one above...e.g. "it went to be done at him" or osmething

**Comment by Lguyogiro** [comment]
Okay, coming back to this: _y_ is _īn_ with the n deleted. so here it is "cuando le acontecía **esto** ..."

**Comment by ftyers** [comment]
How is this?

![imatge](https://github.com/ftyers/UD_Classical_Nahuatl-FloCo/assets/449545/c6625760-ae65-41f0-a00c-89cbc527a4c4)

**Comment by Lguyogiro** [comment]
hmm, here I would've had `nsubj(mochiuh, y)`, "cuando sucedió que esto se le hizo" or something?  and actually, looking at it that way, maybe `csubj(oiuh, mochiuh)`?

**Comment by ftyers** [comment]
Like this? 

![imatge](https://github.com/ftyers/UD_Classical_Nahuatl-FloCo/assets/449545/fa784d0b-0299-40b0-9a18-3d497ee87d09)

(although there is no *in* introducing *ipan mochiuh, y*...)

**Comment by Lguyogiro** [comment]
LGTM

**Comment by ftyers** [comment]
Merged in 4d37cc7.
